code/modules/client/preference_setup/loadout/loadout_head.dm

```diff
@@ -1,9 +1,3 @@
-/datum/gear/head
```

Comment:
I think that removing this datum will affect all the other head ones, due to it being the parent for all others.